prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript DOM Creation</title>
+</head>
+<body>
+  <div id="container">
+
+  </div>
+  <h1>Hello World!</h1>
+  <script>
```

Comment:
Hi Sophia, for this exercise we want to use JavaScript to add the heading element instead of modifying the HTML code.